### DISCUSSION TITLE
500报错并提示【message=心跳过期或GameId错误】

### DISCUSSION THREAD

**Discussion by MuyueJusan** [xfgryujk/blivechat#182] [提问] [answered]
![image](https://github.com/xfgryujk/blivechat/assets/86054388/09136a74-59d8-416a-846d-25317399f3fa)
请问这个是...什么情况，在本地模式出现的

**Comment by xfgryujk** [comment]
你到Cloudflare的线路有点差。如果还能接收弹幕就不用管

**Comment by MuyueJusan** [comment]
> 你到Cloudflare的線有點差。如果能夠接收彈幕就不用管

有时候接受不到或者接收缓慢，请问老师应该如何解决？

**Comment by xfgryujk** [comment, marked as answer]
> > 你到Cloudflare的線有點差。如果能夠接收彈幕就不用管
> 
> 有时候接受不到或者接收缓慢，请问老师应该如何解决？

https://github.com/XIU2/CloudflareSpeedTest

**Comment by MuyueJusan** [comment]
> > > 你到Cloudflare的線有點差。如果能夠接收彈幕就不用管
> > 
> > 
> > 有时候接受不到或者接收缓慢，请问老师应该如何解决？
> 
> https://github.com/XIU2/CloudflareSpeedTest

感谢！！！问题解决了！！！没有超级延迟，弹幕的获取也正常了！！！
